src/webhooks/serializers/index.ts

```diff
@@ -1,3 +0,0 @@
-export * from './webhook-directory-group.serializer';
```

Comment:
As stated elsewhere, all of these were pulled in to `directory-group.serializer`, `directory-user.serializer` and `directory.serializer` respectively.